resources/styles.css

```diff
@@ -211,3 +219,50 @@ input.tab-text {
     width: 20px;
     height: 20px;
 }
+
+
+
+
+html, body, .r_flex_container, #app{
+    height: 100%;
+    display: flex;
+    flex-direction: column;
+    margin: 0;
+}
+.r_flex_container {
+    display:flex;
+    flex-flow: column nowrap;
+}
+
+.r_flex_fixed_child_top {
+    background-color:black;
+    flex:none;
+    margin-bottom: 50px;
+}
+
+.r_flex_fixed_child_bott {
+    background-color:black;
+    flex:none;
+    margin-bottom: 0px;
+}
+
+.r_flex_expand_child {
+    flex:auto;
+    overflow-y:auto;
+}
+
+#left {
+  float:left;
+  width:180px;
+  padding-left: 20px;
+    
+}
+#right {
+  padding-right: 20px;
+  width: 100%;
+  text-align: right;
+}
+
+ul{
```

Comment:
Why do we need this?

Reply:
It seems there are some css files at some node_modules and one of them have rules to <ul> elements that made them have a top-margin which made the problem with right scroll bar. 
I'm not sure why but it seems that your styless.css has preference to these css files and, as I didn't know if  we could change them or if it would be a good idea, I decided to try writting this rule and it did the trick.
Then I realiced that, as the list that was generating troubles has the class collapsible, it would be the same if I put that rule at collapsible (what I did) but I forgot to remove it from ul.

The file I mentioned is.. materialize.min.css, which made a top margin of 7px

So.. I remove that rule!